code/game/objects/items/defibrillator.dm

```diff
@@ -157,7 +157,7 @@
 	if(skill < SKILL_MEDICAL_PRACTICED)
 		user.visible_message(span_notice("[user] fumbles around figuring out how to use [src]."),
 		span_notice("You fumble around figuring out how to use [src]."))
-		var/fumbling_time = SKILL_TASK_AVERAGE - ( SKILL_TASK_VERY_EASY * ( SKILL_MEDICAL_PRACTICED - skill ) ) // 3 seconds with medical skill, 5 without
+		var/fumbling_time = SKILL_TASK_AVERAGE - (SKILL_TASK_VERY_EASY * skill) // 3 seconds with medical skill, 5 without
```

Comment:
```suggestion
		var/fumbling_time = SKILL_TASK_AVERAGE - (2 SECONDS * skill) // 3 seconds with medical skill, 5 without
```
Same thing, but easier to glance at.

Reply:
Yeah I thought so as well, but I decided to keep it consistant with the one earlier in the file and elsewhere. Happy to change it if maintainers think it's better though.

Reply:
No magic numbers

Reply:
it doesnt matter imo